server/src/interfaces/job.interface.ts

```diff
@@ -76,12 +76,12 @@ export enum JobName {
   FACIAL_RECOGNITION = 'facial-recognition',
 
   // library management
-  LIBRARY_SCAN = 'library-refresh',
-  LIBRARY_SCAN_ASSET = 'library-refresh-asset',
-  LIBRARY_REMOVE_OFFLINE = 'library-remove-offline',
-  LIBRARY_CHECK_OFFLINE = 'library-check-offline',
+  LIBRARY_QUEUE_SYNC_FILES = 'library-scan-new',
+  LIBRARY_QUEUE_SYNC_ASSETS = 'library-queue-remove-deleted',
+  LIBRARY_SYNC_FILE = 'library-refresh-asset',
+  LIBRARY_SYNC_ASSET = 'library-remove-deleted',
   LIBRARY_DELETE = 'library-delete',
```

Comment:
Nit: can we use the enum names as the values instead of using new words? To me this is somewhat confusing.